pkg/pkgmgr/dpkg.go

```diff
@@ -459,115 +477,97 @@ func (dm *dpkgManager) unpackAndMergeUpdates(ctx context.Context, updates unvers
 			})).Root()
 	}
 
+	updated = updated.Run(buildkit.Sh("mkdir -p /tmp/debian-rootfs/var/lib/dpkg")).Root()
+
+	// Replace status file in tooling image with new status file with relevant pacakges from image to be patched.
+	// Regenerate /var/lib/dpkg/info files based on relevant pacakges from image to be patched.
+	dpkgdb := updated.Run(
+		llb.AddEnv("PACKAGES_PRESENT_ALL", string(jsonPackageData)),
+		llb.AddEnv("STATUS_FILE", dm.tempStatusFile),
+		llb.Args([]string{
+			`bash`, `-xec`, `
+							set -ex
+
+							json_str=$PACKAGES_PRESENT_ALL
+
+							rm -r /var/lib/dpkg/info
+							mkdir -p /var/lib/dpkg/info
+
+							apt-get clean
```

Comment:
does this delete our packages.txt inside `/var/cache/apt/archives/`? I'm wondering since the download script relies on it when `updates == nil`

Reply:
thats is a good point, it looks like it would delete this file. Im not sure if this is needed anymore, so I will remove the command